twine.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.test_files   = Dir.glob("test/test_*")
 
   s.required_ruby_version = ">= 1.8.7"
-  s.add_runtime_dependency('rubyzip', "~> 0.9.5")
+  s.add_runtime_dependency('rubyzip', "~> 1.1.7")
```

Comment:
Would it be better to ~> 1.1 here? Or did rubyzip change APIs between 1.1 and 1.1.7?